panel/models/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module "json-formatter-js";
```

Comment:
This package has type information, so this shouldn't be needed.

Reply:
Can't seem to find it:

```
panel/models/json.ts:3:27 - error TS7016: Could not find a declaration file for module 'json-formatter-js'. '/Users/philippjfr/development/pyviz_panels/panel/node_modules/json-formatter-js/dist/json-formatter.js' implicitly has an 'any' type.
```

Reply:
Scratch that, they messed up the configuration, so no type info. Also on a related note, looking closer at this library, it doesn't seem to be maintained anymore.

Reply:
Since it doesn't have any upstream runtime dependencies I can live with it not being maintained. Do you think I **should** be concerned?

Reply:
It depends, though (TL;DR) in this case no. Given that it doesn't have dependencies is a good thing and avoids security issues, patching up `package-lock`, etc. Type information can be defined locally, so that's not a big issue. So the only real issue is fixing any potential bug and/or adding new features. If you have to change it, you will have to fork it and continue development on your own. Given that this is JS, some things can be patched up without changing the source code. This is (I presume) a "leaf" feature, i.e. you aren't going to build on top of it, so it's a low risk. To the contrary, in bokeh we depend on canvas2svg, which also isn't maintained anymore and it's a big problem (essentially every new issue is a "won't fix").